nekoyume/Assets/_Scripts/UI/Model/RankCategory.cs

```diff
@@ -0,0 +1,10 @@
+namespace Nekoyume.UI.Module
+{
+    public enum RankCategory
+    {
+        Ability = 0,
+        Stage = 1,
+        Mimisburnnr = 2,
+        Weapon = 3,
+    }
+}
```

Comment:
The location of this file is `.../UI/Model/RankCategory`, but the namespace is `Nekoyume.UI.Module`. Does not match.
And where do you use the `RankCategory` enum? I'm wondering where the location of this enum is good.

Reply:
Seems like it's typo of `Model`. Fixed it.
It is used in `RankPanel` to categorize sorting filter, and I put this enum here because similar type of enum (`ShopSortFilter`) is in the same namespace.